sdk/monitor/azure-monitor-ingestion/src/main/java/com/azure/monitor/ingestion/implementation/Batcher.java

```diff
@@ -118,7 +119,12 @@ private LogsIngestionRequest nextInternal() throws IOException {
         LogsIngestionRequest result = null;
         while (iterator.hasNext() && result == null) {
             Object currentLog = iterator.next();
-            byte[] bytes = serializer.serializeToBytes(currentLog);
```

Comment:
This should continue to use `serializer` as that is meant to handle the custom types that may be provided.